cobalt/browser/metrics/cobalt_metrics_service_client.h

```diff
@@ -31,6 +42,8 @@ class SyntheticTrialRegistry;
 
 namespace cobalt {
 
+constexpr int kStandardUploadIntervalMinutes = 5;
```

Comment:
You can use base::Minutes() directly here (it'll be a TimeDelta IIRC) since there's a constexpr constructor already:
`constexpr auto kStandardUploadInterval = base::Minutes(5);`